templates/one_loop_decay_diagrams.cpp.in

```diff
@@ -0,0 +1,7174 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+#include "@ModelName@_one_loop_decay_diagrams.hpp"
+#include "pv.hpp"
+#include "wrappers.hpp"
+#include "loop_libraries/loop_library.hpp"
+
+namespace flexiblesusy {
+
+inline double Den(double x2, double y2) {
```

Comment:
Please put this function into an anonymous namespace, in order to help the optimizer.

Reply:
Could you please check a681c571b0cd99f0b82100ff5d58e7717446c306?

Reply:
Perfect! :)